conversions/bytes_to_int.py

```diff
@@ -0,0 +1,59 @@
+def bytes_to_int(bytes_var: bytearray, signed: bool, byteorder: str) -> (int):
```

Comment:
Let's follow the inputs, outputs, and exceptions of `int.from_bytes()` exactly...
```suggestion
def bytes_to_int(bytes: bytes, byteorder: str, *, signed: bool=False) -> int:
```